addons/calendar/models/calendar_event.py

```diff
@@ -690,10 +690,14 @@ def create(self, vals_list):
         for values in vals_list:
             # created from calendar: try to create an activity on the related record
             if not values.get('activity_ids'):
-                defaults = self.default_get(['activity_ids', 'res_model_id', 'res_id', 'user_id'])
+                defaults = self.default_get(['activity_ids', 'res_model_id', 'res_id', 'user_id', 'partner_ids'])
                 res_model_id = values.get('res_model_id', defaults.get('res_model_id'))
                 res_id = values.get('res_id', defaults.get('res_id'))
                 user_id = values.get('user_id', defaults.get('user_id'))
+                # Bug with quick_create values are added after create -> no attendee
+                # (only add attendee if there is more than just the organizer for legacy purpose)
+                if len(defaults.get('partner_ids')) > 1:
```

Comment:
from my tests, it is always a length of 1

for example if I do:

```
> self.env['calendar.event'].with_context(default_partner_ids=[1, 2, 3]).default_get(['partner_ids'])
{'partner_ids': [(6, 0, [1, 2, 3])]}
```

Reply:
:ok_hand: